projects/notebooks/visualizations/rhods-notebooks-performance/store/parsers.py

```diff
@@ -343,7 +345,7 @@ def _parse_env(dirname, test_config):
             JENKINS_ARTIFACTS=f"https://{jenkins_instance}/{jenkins_job}/{build_number}/artifact/run/{jumphost}/{base_path}/{from_env.test.test_path}"
         )
 
-    if test_config.get("export_artifacts.enabled"):
+    if test_config.get("export_artifacts.enabled") and not outside_test_env:
```

Comment:
can you turn this into:
```suggestion
    if test_config.get("export_artifacts.enabled") and from_env.test.ci_engine is not None:
```

and maybe above, replace:
```
if ansible_env.get("PERFLAB_CI") == "true":
by
if from_env.test.ci_engine == "PERFLAB_CI"
and same above:
if from_env.test.ci_engine == "OPENSHIFT_CI"
```